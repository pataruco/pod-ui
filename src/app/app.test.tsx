import { shallow } from 'enzyme';
import React from 'react';

import App from './';

describe('App', () => {
  it('renders', () => {
    const result = shallow(<App />);
    expect(result).toMatchSnapshot();
  });
});
