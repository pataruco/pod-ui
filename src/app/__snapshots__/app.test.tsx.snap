// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="app"
>
  <h1>
     Hola, World! 
  </h1>
</div>
`;
